package/Dockerfile

```diff
@@ -39,7 +39,7 @@ ENV KUSTOMIZE_VERSION v3.5.4
 # System charts minimal version
 ENV CATTLE_FLEET_MIN_VERSION=0.3.500
 ENV CATTLE_RANCHER_OPERATOR_MIN_VERSION=0.1.300
-ENV CATTLE_RANCHER_WEBHOOK_MIN_VERSION=0.1.0-beta900
+ENV CATTLE_RANCHER_WEBHOOK_MIN_VERSION=0.1.000
```

Comment:
I think master should be on https://github.com/rancher/webhook/releases/tag/v0.2.0-alpha01

Reply:
oh, good point